__tests__/app.test.js

```diff
@@ -137,15 +137,55 @@ it('should return an array of articles' , () => {
     })  
   }) 
 
+  it.only('should filter the articles by topic', () => {
+    return request(app)
+    .get('/api/articles?topic=cats')
+    .expect(200)
+    .then(({ body }) => {
+        body.articles.forEach((article) => {
```

Comment:
Don't forget we need a length check any time we use a forEach to prevent a false pass!